### PR TITLE
Hotfix: 63 git hub templates

### PR DESCRIPTION
**Pull request by damian-romero**
See #63

Add the latest version of PR and ISSUE templates from LD_Cool-P
Fix links to work with this repository

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**Reproduction Steps**
+<!-- Steps to reproduce the behavior -->
+
+**Outputted Messages**
+<!-- Provide the full message log if possible or a subset that includes a few lines before the failure -->
+
+**Version information (please complete the following, default in brackets)**
+ - ReQUIAM_csv version: [e.g. `0.11.0`] 
+ - `pandas` version: [e.g., 1.2.1]
+ - `numpy` version: [e.g., 1.19.5]
```
Comment (astrochun) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
**Version information**
<!-- Update with tagged version or branch full name-->
[requirements](https://github.com/ualibraries/ReQUIAM_csv/blob/v<version>/requirements.txt)
[requirements](https://github.com/ualibraries/ReQUIAM_csv/blob/<branch>/requirements.txt)
```

**Review by astrochun — CHANGES_REQUESTED** [review]
Changes look good but see one suggestion below.

**Review by astrochun — COMMENTED** [review]
Looks good, but here's the problem, the branch name is incorrect now that I look at it. It won't easily allow for versioning via `git-flow` so I have to fix that. Please be consistent with branch names within `git-flow`

**Review by astrochun — APPROVED** [review]

**Comment by damian-romero** [comment]
Fixed in #70

**Pull request closed without merging**